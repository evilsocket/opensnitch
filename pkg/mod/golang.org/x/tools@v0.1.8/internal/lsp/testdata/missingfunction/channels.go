package missingfunction

func channels(s string) {
	undefinedChannels(c()) //@suggestedfix("undefinedChannels", "quickfix")
}

func c() (<-chan string, chan string) {
	return make(<-chan string), make(chan string)
}
