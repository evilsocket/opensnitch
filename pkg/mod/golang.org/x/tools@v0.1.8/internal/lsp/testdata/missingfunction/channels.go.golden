-- suggestedfix_channels_4_2 --
package missingfunction

func channels(s string) {
	undefinedChannels(c()) //@suggestedfix("undefinedChannels", "quickfix")
}

func undefinedChannels(ch1 <-chan string, ch2 chan string) {
	panic("unimplemented")
}

func c() (<-chan string, chan string) {
	return make(<-chan string), make(chan string)
}

